src/bot.jl

```diff
@@ -1,55 +1,66 @@
-export BotConfig, snoop_bot, snoop_bench, snoopi_bot, snoopc_bot, snoopi_bench, snoopv_bench
+export BotConfig, snoop_bot, snoop_bench
 
 if VERSION <=  v"1.1"
-    isnothing(x) = x == nothing
+    isnothing(x) = x === nothing
 end
 if VERSION <=  v"1.2"
     Base.print(io, ::Nothing) = Base.print(io, "")
     Base.print(::Base.GenericIOBuffer{Array{UInt8,1}}, ::Nothing) = Base.print(io, "")
     Base.string(::Nothing) = ""
 end
+
 ################################################################
 const UStrings = Union{AbstractString,Regex,AbstractChar}
 
 """
-    BotConfig(package_name ; blacklist, os, else_os, version, else_version, package_path, precompiles_rootpath, subst, tmin)
-
-Construct a SnoopCompile bot configuration. `package_name` is the name of the package. This object is fed to the `snoopi_bot`.
+    BotConfig(package_name::AbstractString ; blacklist, os, else_os, version, else_version, package_path, precompiles_rootpath, subst, tmin)
 
-# Arguments:
-- `package_name::AbstractString`
+Construct a SnoopCompile bot configuration. `package_name` is the name of the package. This object is passed to [`snoop_bot`](@ref)
+and [`snoop_bench`](@ref).
 
 You may supply the following optional **keyword** arguments:
 
-- `blacklist` : A vector of of Strings (or RegExp) to remove some precompile sentences
+- `blacklist` : A vector of of Strings (or RegExp) to remove some precompile statements
 
-- `exhaustive`: passing `true` enables exhaustive testing of each precompilation signature, however, it slows down the processing stage. Defaults to `false`. Use this feature if SnoopCompile benchmark or your CI fails. This feature also prints the errors, which can help to find the `blacklist`s indivudally and disble `exhaustive` for the next runs.
+- `exhaustive`: passing `true` discards precompile statements that cannot be `eval`ed.
+  However, it slows down the processing stage.
+  Defaults to `false`. Use this feature if SnoopCompile benchmark or your CI fails.
+  This feature also prints the errors, which you can fix by `blacklist`ing the offending statements
+  and then disabling `exhaustive` for future runs.
 
-- `os`: A vector of of Strings (or RegExp) to give the list of os that you want to generate precompile signatures for. Each element will call a `Sys.is\$eachos()` function.
+- `os`: A vector of of Strings (or RegExp) to support with precompile statements.
+  Each element will call a `Sys.is\$eachos()` function.
```

Comment:
We can probably remove this notice because now we have `standardize_osname` which makes it hard to give a wrong OS name.
```suggestion

```